### DISCUSSION TITLE
Problem with login

### DISCUSSION THREAD

**Discussion by novakadam3** [chemokita13/beReal-api#3] [Q&A]
Hi, your api looks great, I would like to use it.
I am trying to send myself OTP code in postman. My url looks like this `https://berealapi.fly.dev/login/send-code?phone=+420123456789` and i keep getting 500 OTP not sent.

Does anyone know what I did wrong? Do i need to add something more to my POST request?

**Comment by chemokita13** [comment]
Hi! I think the BeReal server is down or maybe the changed the route, I will work on it!
However the other routes are working perfectly.

**Comment by chemokita13** [comment]
Where are you doing your request? I have tried on the onpage swagger api and doesnt work, but in my [frontend proyect](https://github.com/chemokita13/berealgate) is allready working. I let you a piece of an example code curently working extrated from [here](https://github.com/chemokita13/BeRealGate/blob/main/src/app/login/page.tsx):
```typescript
const { data, status }: { data: ApiResponse; status: number } =
                await axios.post("login/send-code", {
                    phone: phone,
                    Headers: {
                        "Access-Control-Allow-Origin": "*", // Required for CORS support to work
                        cors: "*",
                    },
                });
```

**Reply by novakadam3** [reply]
I am just trying this request in my local enviroment with postman. And in your [app](https://berealgate.vercel.app), it sends the code, but its unable to verify :/. Are you sure that it works with +420 before the phone number?

**Reply by chemokita13** [reply]
I think the issue has solved!

**Reply by khodgson2001** [reply]
Hi, I don't think +44 numbers are supported? I'm getting errors on the API and in your app when I try to use a GB number (+44 and then 10 digits after)

**Reply by chemokita13** [reply]
> Hi, I don't think +44 numbers are supported? I'm getting errors on the API and in your app when I try to use a GB number (+44 and then 10 digits after)

And maybe using the login/send-code-vonage endpoint?

**Reply by khodgson2001** [reply]
Also doesn't work - I've tried doing it via Postman and then also on the SwaggerUI to test and both of those end up returning the 400 error

**Comment by chemokita13** [comment]
@khodgson2001 Let me work on it then!

**Reply by khodgson2001** [reply]
Awesome! Thanks :D
